Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```
@@ -704,6 +706,106 @@
+    access: [["Command"]]
+  - type: Lock
+    unlockOnClick: false
+
+- type: entity
+  parent: ComputerStationTeleportersControl
+  id: ComputerStationTeleportersControlFilled
+  suffix: Autofilled, Nanotrasen Network
+  components:
+  - type: StationTeleporterConsole
+    autoLinkKey: Nanotrasen
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)